airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -214,9 +214,13 @@ def consume_logs(*, since_time: Optional[DateTime] = None, follow: bool = True)
                 for line in logs:
                     timestamp, message = self.parse_log_line(line.decode('utf-8'))
                     self.log.info(message)
-            except BaseHTTPError:  # Catches errors like ProtocolError(TimeoutError).
+            except BaseHTTPError:
                 self.log.warning(
-                    'Failed to read logs for pod %s',
+                    "Reading of logs was interrupted, likely due to a connection issue. "
+                    "Will retry momentarily.",
```

Comment:
Shoudl we at least add the error message - not traceback - here in parenthesis (without it, It feels completely blind guessing).

I think that would strike the right balance

Reply:
ok take look